subcommands/events/create.go

```diff
@@ -0,0 +1,56 @@
+package events
+
+import (
+	"io/ioutil"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	cmd.AddCommand(&cobra.Command{
+		Use:   "mk-push <label> <url>",
```

Comment:
Customer may want to provide some auth token along with a URL to push to?

Reply:
no - this is done in standard webhook fashion where the sender will include a JWT that can be verified by the receiver. This is described in the "Webhook Security" section of the design doc and will be covered in the user facing docs as well.